src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -30,6 +31,8 @@
     public static final String MESSAGE_SUCCESS = "New person added: %1$s";
     public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
 
+    public static final String MESSAGE_AUTOMATIC_LIST = "Listing of addressbook after new person added";
+
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling. Why not group `MESSAGE_AUTOMATIC_LIST` with the other messages above?